lib/ocamlorg/module_map.ml

```diff
@@ -0,0 +1,130 @@
+module StringMap = Map.Make (String)
+
+type kind =
+  | Module
+  | Page
+  | LeafPage
+  | ModuleType
+  | Argument
+  | Class
+  | ClassType
+  | File
```

Comment:
```suggestion
  | Module
  | Page
  | Leaf_page
  | Module_type
  | Argument
  | Class
  | Class_type
  | File
```

Same as above, to stay in line with the conventions.

Reply:
FWIW, it looks like this is matching odoc's conventions - see [here](https://github.com/ocaml/odoc/blob/91f6310967e64f3fa88445f3daf2fea2acc0bb49/src/document/url.ml#L80-L88)